v8.03 w/ ability to customize C filters

EZhelp opened the pull request:
When "Enhance dialog clarity" is checked, a pop up table allows the user to customize the C filters for their specific needs. Although this seems to be working, it is a beta and more testing is needed. 

* Since the original option had 5 filters, this modification has 5 adjustable filters. However, since they are adjustable, maybe 3 would be enough? If so, maybe 80, 2500 and 5000hz as defaults?  The current range allowed is 50-12000. Please double check that the ranges for each item in the table are valid and do not interfere with other code.
* Updated: I now have a version where the grid is 1 row lower and centered on the screen for a cleaner look.
:-)  
CPC

ObsessiveCompulsiveAudiophile commented:
Thanks for the addition. I am sure some people will make use of that but this is not a common request so cannot go into default Acoustica.

EZhelp commented:
Hello Serkan 

Cantrellpc from the AVS forum here.
Please take a moment to reconsider adding this modification to the main branch.  This turns the Enhanced Dialog Clarity option into a very power and useful tool for a wider range of users.  It makes your EDC option adjustable to the user’s specific needs.
Users have requested a way to correct specific issues with their C channel in the past, this versatile mod addresses some of those issues and has received positive feedback on the AVS forum as well as multiple messages from users thanking and giving examples on how they were able to use it to clean up their specific persistent dialog issues. Here are just two posts from AVS #9853 and #9935 where I introduced it. Users from the facebook group have even reached out with thanks.

You may not have given much thought to this request at first, as you were dealing with a troll at the same time. (The troll may have been from one of your commercial competitors? (who knows?))  In any case, do not let the troll win.

Please take a serious look at this modification and reconsider adding it to the main branch.  The latest version is 0.0.4 in the EZhelp fork which has .5 step sliders for the PEQ gain controls.

Thank you and I hope all is well.
Cantrellpc
